python/src/ir.cc

```diff
@@ -1906,7 +1906,15 @@ void init_triton_ir(py::module &&m) {
                 setupTritonDiagnosticHandler(context);
             if (failed(self.run(mod.getOperation())))
               throw std::runtime_error("PassManager::run failed");
+
+            if (!repro_suffix.empty() && !reproducerPath.empty() &&
+                reproducerPath != "-" &&
+                llvm::sys::fs::copy_file(reproducerPath,
+                                         reproducerPath + repro_suffix)) {
+              throw std::runtime_error("PassManager::run failed (repro temp)");
+            }
           },
+          py::arg("mod"), py::arg("repro_suffix") = "",
```

Comment:
why do you need that?

Reply:
I believe I needed to add this because of the default param for the `repro_suffix`, I can test out if it works just fine without default paramming it.

Reply:
Dropped